src/components/pacbio/PacbioRunPlateItem.vue

```diff
@@ -1,23 +1,43 @@
 <template>
-  <Plate96SVG :height="'100%'" :width="'100%'">
-    <well v-for="(well, key) in plateMap.wells" :key="key" v-bind="well" @alert="alert"> </well>
-  </Plate96SVG>
+  <flagged-feature name="enable_custom_modal">
+    <template #disabled>
+      <Plate96SVG :height="'100%'" :width="'100%'">
+        <b-well v-for="(well, key) in plateMap.wells" :key="key" v-bind="well" @alert="alert">
+        </b-well>
+      </Plate96SVG>
+    </template>
+    <Plate96SVG :height="'100%'" :width="'100%'">
+      <well
+        v-for="(well, key) in plateMap.wells"
+        :key="key"
+        v-bind="well"
+        @alert="alert"
+        @click="onWellClick"
+      >
+      </well>
+    </Plate96SVG>
+    <WellEdit ref="modal" class="modal" :position="selectedWellPosition" @alert="alert"></WellEdit>
+  </flagged-feature>
 </template>
 
 <script>
 import Plate96SVG from '@/components/svg/Plate96SVG'
 import PlateMap from '@/config/PlateMap'
+import BWell from '@/components/pacbio/PacbioRunWellItemBS'
```

Comment:
I would maybe rename this to be PacbioRunWellItemBootstrap rather than PacbioRunWellItemBS? It just removes any ambiguity.

Reply:
Done 👍